cmd/keycloakb/keycloak_bridge.go

```diff
@@ -594,6 +597,9 @@ func main() {
 		managementSubroute.Path("/realms/{realm}/clients/{clientID}/roles").Methods("GET").Handler(getClientRolesHandler)
 		managementSubroute.Path("/realms/{realm}/clients/{clientID}/roles").Methods("POST").Handler(createClientRolesHandler)
 
+		//groups
+		managementSubroute.Path("/realms/{realm:[a-zA-Z0-9_-]+}/groups").Methods("GET").Handler(getGroupsHandler)
```

Comment:
the regexp is not needed anymore here. We manage it in http.go

Reply:
DONE